app/overrides/controllers/decidim/consultations/question_votes_controller.rb

```diff
@@ -17,21 +17,15 @@ def delegation
     @delegation ||= Decidim::ActionDelegator::Delegation.find_by(id: params[:decidim_consultations_delegation_id])
   end
 
-  def enforce_permission_to_unvote(subject, extra_context = {})
+  def enforce_permission_to_unvote
```

Comment:
this if would be better off replaced by two different entry points. That is, having a separate controller with the action to unvote a delegation, while leaving the `Decidim::Consultations` controller untouched. The former would use the `:unvote_delegation` permission action while the latter would use the regular `:unvote`.

Reply:
A new `Decidim::ActionDelegator::DelegationVotesController` should do it